lib/ex_sdp/attribute/fmtp.ex

```diff
@@ -32,6 +32,9 @@ defmodule ExSDP.Attribute.FMTP do
                 # OPUS
                 :maxaveragebitrate,
                 :maxplaybackrate,
+                :sprop_maxcapturerate,
+                :maxptime,
+                :ptime,
```

Comment:
That's pretty interesting because [RFC 7587 (sec 7)](https://www.rfc-editor.org/rfc/rfc7587.html#section-7) says:

```
      The OPTIONAL media type parameters "ptime" and "maxptime" are
      mapped to "a=ptime" and "a=maxptime" attributes, respectively, in
      the SDP.

     The OPTIONAL media type parameters "maxaveragebitrate",
      "maxplaybackrate", "stereo", "cbr", "useinbandfec", and "usedtx",
      when present, MUST be included in the "a=fmtp" attribute in the
      SDP, expressed as a media type string in the form of a semicolon-
      separated list of parameter=value pairs (e.g.,
      maxplaybackrate=48000).  They MUST NOT be specified in an SSRC-
      specific "fmtp" source-level attribute (as defined in Section 6.3
      of [RFC5576]).
      
 ```
 
 so `ptime` and `maxptime`  shouldn't be present in FMTP. We also implemented them here https://github.com/membraneframework/ex_sdp/blob/master/lib/ex_sdp/attribute.ex#L35-L36

There are also some examples

```
       m=audio 54312 RTP/AVP 101
       a=rtpmap:101 opus/48000/2
       a=fmtp:101 maxplaybackrate=16000; sprop-maxcapturerate=16000;
       maxaveragebitrate=20000; stereo=1; useinbandfec=1; usedtx=0
       a=ptime:40
       a=maxptime:40
```
 
However, looking at mozilla implementation of SDP they have `maxptime` and `ptime` both in FMTP and as separate attributes so I think we can follow them :wink: 

https://docs.rs/webrtc-sdp/0.3.9/webrtc_sdp/attribute_type/struct.SdpAttributeFmtpParameters.html
https://docs.rs/webrtc-sdp/0.3.9/webrtc_sdp/attribute_type/enum.SdpAttribute.html#variant.Ptime

Reply:
Hey @mickel8, Thank you! 

So I brought in that suggestion, and two test cases. One that validates spaces and one that validates the new fmtp parameters. Let me know if that works. 🙂